docs/KVSet/KVSet.v1.0.0.schema.json

```diff
@@ -0,0 +1,97 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "$id": "http://azconfig.io/schemas/KVSet/v1.0.0/KVSet.json",
+  "title": "An Azure App Configuration schema for importing/exporting using 'appconfig/kvset' format",
+  "type": "object",
+  "required": [
+    "items"
+  ],
+  "properties": {
+    "items": {
+      "$id": "#/properties/items",
+      "type": "array",
+      "title": "Items",
+      "description": "An array of key-values in the Azure App Configuration store.",
+      "items": {
+        "$ref" : "#/$defs/kv"
+      }
+    }
+  },
+  "$defs": {
+    "kv": {
+      "type": "object",
+      "title": "Key value pair",
+      "description": "An individual key-value pair and associated metadata in the 'appconfig/kvset' format.",
```

Comment:
Remove "pair". We decided to call it just "key-value" everywhere. It will be consistent with all our docs.